src/Elm/Parser.elm

```diff
@@ -40,14 +39,4 @@ When parsing fails, the result will contain a list of errors indicating what wen
 -}
 parseToFile : String -> Result (List DeadEnd) File
 parseToFile input =
-    case Combine.runParser (withEnd file) emptyState input of
-        Ok ( _, fileContents ) ->
-            Ok fileContents
-
-        Err s ->
-            Err s
-
-
-withEnd : Parser State File -> Parser State File
-withEnd p =
-    p |> Combine.ignore end
+    Core.run (file |. Core.end) input
```

Comment:
Is there a reason that we don't put `Core.end` directly in the definition of `file`? :thinking:

Reply:
seems reasonable to me.